Cargo.toml

```diff
@@ -18,3 +18,14 @@ members = [
     "test-dynamic-plugin-rust",
     "test-dynamic-plugins",
 ]
+
+# To apply the following lint settings to a workspace member,
+# set "lints.workspace" to true in the member's Cargo.toml.
+[workspace.lints.clippy]
+redundant_field_names = "deny"
+needless_return = "deny"
+needless_borrow = "deny"
+comparison_to_empty = "deny"
+single_char_pattern = "deny"
+unused_enumerate_index = "deny"
+useless_format = "deny"
```

Comment:
Will it put the job in warning state if the is another warning from clippy which is not in this list ?

Reply:
it prints warning in the logs but there's no link with GitHub yet